pkg/schedule/checker/learner_checker.go

```diff
@@ -36,13 +36,13 @@ func NewLearnerChecker(cluster sche.CheckerCluster) *LearnerChecker {
 }
 
 // Check verifies a region's role, creating an Operator if need.
-func (l *LearnerChecker) Check(region *core.RegionInfo) *operator.Operator {
-	if l.IsPaused() {
+func (c *LearnerChecker) Check(region *core.RegionInfo) *operator.Operator {
```

Comment:
but commit message is [reduce rand NewSource](https://github.com/tikv/pd/pull/8677/commits/34852ed15d51f790ca7a8db2523e7172800a3b42)

Reply:
need to merge #8675 first.